.github/workflows/ci.yml

```diff
@@ -12,14 +11,23 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
```

Comment:
Checkout and setup node are both version 6 now. It doesn't look like there are any breaking changes for `checkout@v6`. There are some breaking changes with `setup-node@v6`. Could we try and update these latest versions? If we're already testing and updating this, then might as well go with the newest versions out at the time of release.